### PR TITLE
Pausing file watch on BigSync, updating threshold on bigsync fallover

### PR DESCRIPTION
**Pull request by joelgriffith**

### CODE REVIEW

File: bin/local.js [hunk 1]
```diff
@@ -6,47 +6,47 @@
...
+    
+    stopFileWatch();
     bigSync(onBigTransferDone);
-    isPaused = true;
 }
 
 function onBigTransferDone() {
     if (config.debug) {
         console.log('['+ config.hostname +'] Received large change');
+        startFileWatch();
```
Comment (chrisdopuch) [inline bin/local.js]: Looks to me like you only startFileWatch again if config.debug is true? Don't we want that in either case?

**Comment by chrisdopuch** [comment]
I like it, looks good. 

Just something to think about, I wonder if it’s possible to miss a file change between when the big sync kicks off, and when it finishes.

File: bin/local.js (hunk 1 above)
Comment (joelgriffith) [inline bin/local.js]: Good catch... that'd suck.

**Comment by chrisdopuch** [comment]
Are we gonna kill this for now @joelgriffith ?
Probably should keep the branch though, and revisit this problem later

**Comment by joelgriffith** [comment]
Closing, will revisit once we get `2.0.0` out.

**Pull request closed without merging**